src/test/java/com/powsybl/openloadflow/sa/WoodburyDcSecurityAnalysisWithActionsTest.java

```diff
@@ -0,0 +1,454 @@
+                new PhaseTapChangerTapPositionAction("pstRelChange", "PS1", true, -1));
+        List<OperatorStrategy> operatorStrategies = List.of(
+                new OperatorStrategy("strategyTapAbsChange", ContingencyContext.specificContingency("L1"), new TrueCondition(), List.of("pstAbsChange")),
+                new OperatorStrategy("strategyTapRelChange", ContingencyContext.specificContingency("L1"), new TrueCondition(), List.of("pstRelChange")));
+
+        SecurityAnalysisResult result = runSecurityAnalysis(network, contingencies, monitors, securityAnalysisParameters,
+                operatorStrategies, actions, ReportNode.NO_OP);
+
+        assertNotNull(result);
+
+        OperatorStrategyResult resultAbs = getOperatorStrategyResult(result, "strategyTapAbsChange");
+        BranchResult brAbsL2 = resultAbs.getNetworkResult().getBranchResult("L2");
+        BranchResult brAbsPS1 = resultAbs.getNetworkResult().getBranchResult("PS1");
+
+        OperatorStrategyResult resultRel = getOperatorStrategyResult(result, "strategyTapRelChange");
+        BranchResult brRelL2 = resultRel.getNetworkResult().getBranchResult("L2");
+        BranchResult brRelPS1 = resultRel.getNetworkResult().getBranchResult("PS1");
+
+        // Apply contingency by hand
+        network.getLine("L1").getTerminal1().disconnect();
+        network.getLine("L1").getTerminal2().disconnect();
+        // Apply remedial action
+        network.getTwoWindingsTransformer("PS1").getPhaseTapChanger().setTapPosition(0);
+        OpenLoadFlowParameters.create(parameters).setLowImpedanceBranchMode(OpenLoadFlowParameters.LowImpedanceBranchMode.REPLACE_BY_MIN_IMPEDANCE_LINE);
+
+        loadFlowRunner.run(network, parameters);
+
+        // Compare results on the line L2
+        assertEquals(network.getLine("L2").getTerminal1().getP(), brAbsL2.getP1(), LoadFlowAssert.DELTA_POWER);
+        assertEquals(network.getLine("L2").getTerminal2().getP(), brAbsL2.getP2(), LoadFlowAssert.DELTA_POWER);
+        assertEquals(network.getLine("L2").getTerminal1().getP(), brRelL2.getP1(), LoadFlowAssert.DELTA_POWER);
+        assertEquals(network.getLine("L2").getTerminal2().getP(), brRelL2.getP2(), LoadFlowAssert.DELTA_POWER);
+        // Compare results on the t2wt PS1
+        assertEquals(network.getTwoWindingsTransformer("PS1").getTerminal1().getP(), brAbsPS1.getP1(), LoadFlowAssert.DELTA_POWER);
+        assertEquals(network.getTwoWindingsTransformer("PS1").getTerminal2().getP(), brAbsPS1.getP2(), LoadFlowAssert.DELTA_POWER);
+        assertEquals(network.getTwoWindingsTransformer("PS1").getTerminal1().getP(), brRelPS1.getP1(), LoadFlowAssert.DELTA_POWER);
+        assertEquals(network.getTwoWindingsTransformer("PS1").getTerminal2().getP(), brRelPS1.getP2(), LoadFlowAssert.DELTA_POWER);
+    }
+
+    @Test
+    void testSaDcPhaseTapChangerTapPositionChangeWithConnectivityBreak() {
+        Network network = PhaseControlFactory.createNetworkWith3Buses();
+
```

Comment:
This is a strange network to test phase shifter action because there is no circuit parallel to the phase shifter.
Actions on the phase shifter are supposed to NOOP no ?

Reply:
I got it. The slack distribution fails.
Can you please fix the test to have correct power values ?
That may help have the test succeed also for normal mode.

Reply:
Indeed, thank you! I've added a line parallel to the PST, to make the TU more consistent. 

About slack distribution, it seems correct in fast DC SA. Here, the generator had a Pmax of 150, so it was not able to go up to 200MW of production after distribution. By changing the Pmax of the generator, the distribution is done correctly.

Slack distribution in normal mode should be fixed in another PR :)

Reply:
@p-arvy  your test was interesting because it helped me find two issues that I plan to address in separate PRs:
      - SlackDistribtionFailuerBheaviour FAIL mode is not honnored by the DC engine
      - The normal DC mode behaves wrongly when a load is lost because of a connectivyt change.

But I wonder why you made a test with unbalanced power and in a mode where the remaining power is left on a slack bus.
If you don't force the slack bus in the test, the value of power in the branches is not really determined because it depends on the slack bus determination algorithm.

Reply:
I reopen. This point was not adressed: If the slack is not fully distributed, and if you don't force the slack bus in the test, the value of power in the branches is not really determined because it depends on the slack bus determination algorithm.

Reply:
I'm not sure to get the point. Why is that a problem if power in branches depends on slack distribution ?

Reply:
It depends on slack bus location. So the test may break if the default algorithm to define the slack bus changes. If you decide to make a test on an unbalanced network with assertions that depend on slack bus location it is better to force the slack bus location in the test.